### DISCUSSION TITLE
用玩客云做旁路由科学上网速度奇慢

### DISCUSSION THREAD

**Discussion by Plutonium141** [coolsnowwolf/lede#10661] [Bugs / 问题反馈]
前一阵子把手里的玩客云刷成openwrt了，固件是自己刚编译的R22.12.1内核版本是5.10.158，用起来发现科学上网速度慢 油管视频速度也就3，4千，一开始怀疑是节点不行，机场换了几家也没有改善，随后用客户端测速 速度并不慢，之后发现是bbr加速导致的，关闭后速度有明显改善。

后来没多久速度又掉下去，发现bbr加速没有启用但还是处于运行中，原因是passwall重启后会自动启动bbr加速（不知道为什么 也许是bug），无奈只好将bbr卸载掉

不清楚是不是我弄错了 或者是开了lan口的ip伪装的原因，发出来希望能给其他人避个坑

还有我记得编译时勾选了squashfs的固件啊，怎么就只出来个ext4的 难道是玩客云不支持squashfs？

**Comment by JHB-L** [comment]
老哥，我用的是R22.12.1内核版本是5.9，设置之后Google什么都行，就是YouTube上不去，你的有这个问题吗？
如果chrome浏览器内设置DNS为8.8.8.8的话，就没问题，但是会导致学校校园网登陆特定网页（知网等）不行了，太头疼了

**Reply by Plutonium141** [reply]
那就是openwrt的dns没配置好吧

**Reply by JHB-L** [reply]
<img width="367" alt="PixPin_2024-04-27_20-16-49" src="https://github.com/coolsnowwolf/lede/assets/79003209/0a0523fe-465b-4698-b9aa-8d8414b79cd4">
<img width="454" alt="PixPin_2024-04-27_20-17-04" src="https://github.com/coolsnowwolf/lede/assets/79003209/d8eaafcb-2044-47c2-9123-90ac83d19898">
<img width="573" alt="PixPin_2024-04-27_20-17-21" src="https://github.com/coolsnowwolf/lede/assets/79003209/91152fe9-83f8-4a64-8083-030151b9f2e8">
<img width="590" alt="PixPin_2024-04-27_20-17-31" src="https://github.com/coolsnowwolf/lede/assets/79003209/3251aaac-ee04-4b47-b4fa-edfc9ad5a7a6">
我这样设置的，其他都默认，电脑是网关DNS都指向玩客云地址，smartdns什么都没动。今天看教程试了试走AdGuard Home，能同时用了，但非常卡。我要是用chrome浏览器内设置DNS为8.8.8.8的话，YouTube速度很快，就是校园网进特定网址有问题。
应该怎么设置

**Reply by Plutonium141** [reply]
看着没什么问题啊 有ipv6环境吗也许是ipv6的问题，再试试passwall打开chinadns-ng如果没用就关了

**Reply by JHB-L** [reply]
不行，难搞

**Reply by Plutonium141** [reply]
dnsmasq的端口是53吗  是不是开了其他插件的dns劫持或者转发什么的？

**Reply by JHB-L** [reply]
发现好像是电脑问题，手机就可以。今天又试了openclash，也是一样的问题，但好像刚开始openclash可以访问YouTube，再后来就不行了，刚试了ssr也不行。刚刚又设置了一下，手机只能加载图片，看不了，openclash界面显示通了

<img width="887" alt="PixPin_2024-04-28_21-05-34" src="https://github.com/coolsnowwolf/lede/assets/79003209/5cb8f629-b2fc-46cb-b8ac-6dd9a01fe1b0">
<img width="992" alt="PixPin_2024-04-28_20-57-13" src="https://github.com/coolsnowwolf/lede/assets/79003209/45162576-2b17-4f3a-93f5-d67e1cfc4c6d">
<img width="494" alt="PixPin_2024-04-28_21-04-50" src="https://github.com/coolsnowwolf/lede/assets/79003209/e06ab813-4a41-4a57-ab26-8c34d9cab172">
